src/MEDS_tabular_automl/scripts/tabularize_static.py

```
@@ -150,6 +160,7 @@ def compute_fn(shard_df):
                 agg=agg,
                 feature_columns=feature_columns,
                 shard_df=shard_df,
+                label_df=label_df,
```

Ruff (0.8.2)

Function definition does not bind loop variable `label_df`

(B023)